third_party/terraform/tests/resource_cloudiot_device_registry_id_test.go

```diff
@@ -0,0 +1,30 @@
+package google
```

Comment:
@danawillow Is `third_party/terraform/tests` the right place for additional tests? I don't see the included in the provider I generate.

Reply:
NVM, found them